fuku/task.py

```diff
@@ -2,12 +2,16 @@
 
 from .module import Module
 from .utils import (
-    StoreKeyValuePair, StorePortPair,
```

Comment:
Hmmm. I guess.

Reply:
Not used, unless u had plans for it? =/

Reply:
Not the removal, but putting everything on individual lines. Hmmph. But okay.